build/build.js

```diff
@@ -105,7 +105,7 @@ class Build {
 
     // All.json (all items in one file)
     all.sort(sort)
-    fs.writeFileSync(path.join(__dirname, '../data/json/All.json'), stringify(all))
+    fs.writeFileSync(path.join(__dirname, '../data/json/All.json'), stringify(all, 2))
```

Comment:
Why would All.json be the single exception, not being minified?

Reply:
because it's the only json:
- not in the build
- used to validate types

Reply:
And validating types is what needs, seemingly, an unminified `All.json` to work correctly. I'm not 100% sure why.